integration_test/gce/gce_testing.go

```diff
@@ -2059,6 +2059,12 @@ func SetupVM(ctx context.Context, t *testing.T, logger *log.Logger, options VMOp
 	return vm
 }
 
+func isOpsAgentUAPPlugin() bool {
```

Comment:
There is already a definition of this in `/integration_test/agents/agents.go` : https://github.com/GoogleCloudPlatform/ops-agent/blob/master/integration_test/agents/agents.go#L1056-L1060

Should we reuse it here ? Or unify the definitions so both can use the same one from a shared place ?

Reply:
I tried unifying them but kept running on cycle dependencies.

Reply:
Another suggested to place the defintion, where it may not face the dependency cycle is next to this similar functions like `IsSLES` ( see https://github.com/GoogleCloudPlatform/ops-agent/blob/2.54.0/integration_test/gce/gce_testing.go ).